pkg/models/libraryelements.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+// LibraryElementKind is used for the kind of library element
+type LibraryElementKind int
+
+const (
+	// PanelElement is used for library elements that are of the Panel kind
+	PanelElement LibraryElementKind = iota + 1
```

Comment:
Why the convention of starting at `iota + 1`?

Reply:
Because I wanted something else than 0. Is there another way?